src/derivers/bip39.ts

```diff
@@ -79,3 +85,33 @@ export async function createBip39KeyFromSeed(
     curve: curve.name,
   });
 }
+
+/**
+ * Create a {@link SLIP10Node} from BIP-39 entropy.
+ *
+ * @param entropy - The BIP-39 entropy.
+ * @param curve - The curve to use.
+ * @returns An object containing the corresponding BIP-39 master key and chain code.
+ */
+export async function createBip39KeyFromEntropy(
+  entropy: Uint8Array,
+  curve: Extract<Curve, { rootKeyPairIsCreatedFrom: 'entropy' }>,
```

Comment:
for the sake of safety, I'd consider narrowing down the types of allowed curves also in `createBip39KeyFromSeed()`, to prevent anybody from using this function with the newly introduced curve, which isn't currently intended